carbonmark-api/test/routes/listings/[id]/get.test.ts

```diff
@@ -15,10 +15,25 @@ describe("GET /listings/:id", () => {
       console.error("/listings/:id get.test.ts setup failed", e);
     }
   });
+
```

Comment:
Might better to abstract this test to 'is an ERC1155' as there could be other non-ICR 1155 projects in the future

Reply:
this is now obsolete due to the subgraph changes where we now have tokenId as part of a listing